main.cpp

```diff
@@ -1,475 +1,581 @@
+
+    //take off
+    bool takeOffComplete(float groundSpeed, bool landingGear, bool armDoorsAndCrossCheck); //returns whether has this instance of Commercial Areoplane completed a successful take off
+    //emergency landing
+    void emergencyLanding(bool enginefailure, float urrentAirSpeed);
+    //inflight entertainment
+    void inflightEntertainment(int passengerSeatNumber, std::string movieSelction);
+};
 
 /*
-Thing 7)
+Thing 5 ) Player
 5 properties:
-    1)
-    2)
-    3)
-    4)
-    5)
+    1) body weight (float)
+    2) sponsor (std:string)
+    3) number of games played (int)
+    4) scoring average (float)
+    5) name (std::string)
 3 things it can do:
-    1)
-    2)
-    3)
- */
+    1) run
+    2) score
+    3) train
+*/
+
+struct Player
+{
+    //body weight (float)
+    float bodyMassKG = 101.34f;
+    //sponsor (std:string)
+    std::string sponsor = "Nike";
+    //number of games played (int)
+    int numGamesPlayed = 104;
+    //scoring average (float)
+    float scoringAverage = 1.78f;
+    //name (std::string)
+    std::string name = "Ronaldo";
+
+    //run
+    float getDistanceRun(int howFast, float howFar, bool startWithLeftFoot);
+    //score
+    void score(bool hasBall, bool inScoringPosition);
+    //train
+    void train(std::string trainingRequired, float fitnessScore);
+};
 
 /*
-Thing 8)
+Thing 6) Coach
 5 properties:
-    1)
-    2)
-    3)
-    4)
-    5)
+    1) name (std::string)
+    2) years experience (int)
+    3) trophies won (int)
+    4) salary (double)
+    5) inspiring (bool)
 3 things it can do:
-    1)
-    2)
-    3)
- */
+    1) give motivational speech
+    2) lose job
+    3) give press conference
+*/
+
+struct Coach
+{
+    //name (std::string)
+    std::string name;
+    //years experience (int)
+    int yearsExperience = 4;
+    //trophies won (int)
+    int numTrophiesWon = 2;
+    //salary (double)
+    double salary = 8078900.99;
+    //inspiring (bool)
+    bool inspiringn = false;
+
+    //give motivational speech
+    void deliverMotivationalSpeech(bool winningStreak);
+    //lose job
+    void getSacked(int numberOfGamesInLosingStreak);
+    //give press conference
+    void givePressConference(bool wonGame);
+};
 
 /*
-Thing 9)
+Thing 7) Fan
 5 properties:
-    1)
-    2)
-    3)
-    4)
-    5)
+    1) name (std::string)
+    2) age (int)
+    3) season ticket holder (bool)
+    4) distance from stadium (double)
+    5) fanclub member (bool)
 3 things it can do:
-    1)
-    2)
-    3)
- */
+    1) buy ticket to game
+    2) cheer
+    3) wear team colours
+*/
+
+struct Fan
+{
+    //name (std::string)
+    std::string name = "John Johnson";
+    //age (int)
+    int age;
+    //season ticket holder (bool)
+    bool hasSeasonTicket = true;
+    //distance from stadium (double)
+    double distanceFromStadiumInKM = 345.78;
+    //fanclub member (bool)
+    bool fanclubMember = true;
+
+    //buy ticket to game
+    void purchseTicket(float ticketPrice, std::string opponentTeam);
```

Comment:
Your function names do not match the plain-english descriptions.
They should match almost 1:1
- [ ] revise